src/box/vy_scheduler.c

```diff
@@ -1116,6 +1117,131 @@ vy_task_write_run(struct vy_task *task, bool no_compression)
 	return -1;
 }
 
+/**
+ * Table of symbols:
+ * R - Worst-case zero-result point lookup cost (in runs);
+ * W_u - number of unfiltered runs;
+ * W_f - number of filtered runs;
+ * t_r - Adopted run_size_ratio_coeff.
+ *
+ * Note:
+ * We should consider the fact that
+ * current range doesn't yet contains newly created run.
+ *
+ * @param lsm
+ * @param range
+ * @param new_run_stmts_count statements count in the newly created after
+ * dump/compaction run
+ * @param W total number of runs in the range
+ * after operation (dump/compaction)
+ * @param total_stmts_count total number of statements in the range
+ * after operation (dump/compaction)
+ * @param skip_n number of runs to skip during the range traverse
+ * as they are compacted into one new run.
+ * @return optimally calculated bloom_fpr
+ */
+double
+vy_task_find_range_bloom_fpr(
```

Comment:
I don't understand the name of this function.

Reply:
Let's move it to `box/vy_stmt.{h, c}` and call it `vy_bloom_optimal_fpr`.

Reply:
See problem above ^^^.